modules/solid_mechanics/src/materials/BackstressRadialReturnStressUpdate.C

```diff
@@ -0,0 +1,65 @@
+#include "BackstressRadialReturnStressUpdate.h"
+#include "RankTwoTensorForward.h"
+
+template <bool is_ad>
+InputParameters
+BackstressRadialReturnStressUpdateTempl<is_ad>::validParams()
+{
+  InputParameters params = RadialReturnStressUpdateTempl<is_ad>::validParams();
+  return params;
+}
+
+template <bool is_ad>
+BackstressRadialReturnStressUpdateTempl<is_ad>::BackstressRadialReturnStressUpdateTempl(
+    const InputParameters & parameters)
+  : RadialReturnStressUpdateTempl<is_ad>(parameters),
+    _backstress(this->template declareGenericProperty<RankTwoTensor, is_ad>("backstress")),
+    _backstress_old(this->template getMaterialPropertyOld<RankTwoTensor>("backstress"))
+{
+}
+
+template <bool is_ad>
+void
+BackstressRadialReturnStressUpdateTempl<is_ad>::initQpStatefulProperties()
+{
+  _backstress[_qp].zero();
+  RadialReturnStressUpdateTempl<is_ad>::initQpStatefulProperties();
+}
+
+template <bool is_ad>
+void
+BackstressRadialReturnStressUpdateTempl<is_ad>::propagateQpStatefulProperties()
+{
+  _backstress[_qp] = _backstress_old[_qp];
+  propagateQpStatefulPropertiesRadialReturn();
+}
+
+template <bool is_ad>
+void
+BackstressRadialReturnStressUpdateTempl<is_ad>::updateState(
+    GenericRankTwoTensor<is_ad> & strain_increment,
+    GenericRankTwoTensor<is_ad> & inelastic_strain_increment,
+    const GenericRankTwoTensor<is_ad> & rotation_increment,
+    GenericRankTwoTensor<is_ad> & stress_new,
+    const RankTwoTensor & stress_old,
+    const GenericRankFourTensor<is_ad> & elasticity_tensor,
+    const RankTwoTensor & elastic_strain_old,
+    bool compute_full_tangent_operator,
+    RankFourTensor & tangent_operator)
+{
+  GenericRankTwoTensor<is_ad> stress_corrected = stress_new - _backstress_old[_qp];
+
+  RadialReturnStressUpdateTempl<is_ad>::updateState(strain_increment,
+                                                    inelastic_strain_increment,
+                                                    rotation_increment,
+                                                    stress_corrected,
+                                                    stress_old,
+                                                    elasticity_tensor,
+                                                    elastic_strain_old,
+                                                    compute_full_tangent_operator,
+                                                    tangent_operator);
+  stress_new = stress_corrected;
+}
+
```

Comment:
I think you need a computeStressFinalize like RadialReturnCreepStressUpdate has.  I thought you had that in CombinedPlasticityStressUpdate when I reviewed this yesterday.

Reply:
I always had computeStressFinalize in only the CombinedPlasticityStressUpdate class. The tests are running correctly this way as intended. 
Still let me know if you think otherwise.